lib/tasks/temporary/attendees.rake

```diff
@@ -0,0 +1,20 @@
+# lib/tasks/temporary/attendees.rake
+namespace :attendees do
+  desc "Migrate sms plans to receive_sms attribute for attendees"
+  task migrate_sms_plans: :environment do
+
+    attendees_with_sms_plans = Attendee.select('attendees.id, attendees.phone, attendees.local_phone, attendees.receive_sms').joins(:plans).where('plans.name = ?', "Yes! SMS!")
+
+    puts <<-TEXT
+    Going to migrate #{attendees_with_sms_plans.to_a.count} attendees from "Yes! SMS!" plan to Attendee receive_sms attribute set to true
+    TEXT
+
+    ActiveRecord::Base.transaction do
+      attendees_with_sms_plans.each do |a|
+        a.update_attribute :receive_sms, true
+        print "."
+      end
+    end
+    puts " All done now!"
+  end
+end
```

Comment:
There are new validations for `local_phone`.  After running the rake task, we can have cases where there are no local phone numbers and cases where there are improperly formatted values of `local_phone`.  Is that an issue?

Reply:
Good question.  The rake task uses the [update_attribute](http://api.rubyonrails.org/classes/ActiveRecord/Persistence.html#method-i-update_attribute) method which skips validations so the rake task will not throw any validation errors and the receive_sms attribute will be updated without a valid local_phone being present. At some point I think we will want to get those who have opted in under the "Yes! SMS!" plan to enter valid local phone numbers (if they have not already). This may have to be done by sending them an email requesting it. Maybe  @neagle could chime in on the best way to get that part done? If we want we could modify the rake task to print a list of   attendee emails that need to be notified.